src/components/Nav/NavList/NavList.js

```diff
@@ -116,6 +116,15 @@ export default class NavList extends Component {
         role="menuitem"
       >
         <div className={`${navItemNamespace}__link`}>
+          <div>
```

Comment:
is this div needed ? if so, should in not be inside your conditional so your not rendering unused elements when the icon isn't present

Reply:
The reason i actually have it in there is so it does render when the icon isn't present. Without it and when the icon isn't present the alignment was all off.

Reply:
I think you need to address the underlying cause of the alignment being off then, having empty divs isn't good practice.